### PR TITLE
Feature/additional stories

### PR DESCRIPTION
**Pull request by matis-dk**

### CODE REVIEW

File: src/styles/scss/shared.scss [hunk 1]
```diff
@@ -22,3 +22,54 @@
   height: 1px;
   overflow: hidden;
 }
+
+.hide-768px {
+  @include breakpoint-s {
+    display: none;
+  }
+}
```
Comment (kasperg) [inline src/styles/scss/shared.scss:30]: As far as I can tell this class is only used once. Should this styling be included with the rest of the class there instead instead of adding a new functional class?

File: src/styles/scss/internal.scss [hunk 2]
```diff
@@ -120,3 +120,11 @@ code {
...
   background-color: #f5f8fa;
 }
+
+.sb-main-padded {
+  padding: 0 !important;
+}
+
+.sb-show-main {
+  padding: 0 !important;
+}
```
Comment (kasperg) [inline src/styles/scss/internal.scss:130]: `!important` should not be used lightly. Could you add a comment explaining why it is necessary?

File: src/stories/list-reservation/ListReservationsAndLoans.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        <div className="list-reservation__material">
+          <div>
+            <Material size="small" animate={true} url={material.image} />
+          </div>
+          <div className="list-reservation__information">
+            <div>
+              <StatusLabel label="bog" status="outline" />
+            </div>
+            <div className="list-reservation__about">
+              <p className="text-header-h4">{material.title}</p>
```
Comment (kasperg) [inline src/stories/list-reservation/ListReservationsAndLoans.tsx:41]: Is there a reason to use a `<p>` tag here for something that could be a headline?

File: src/stories/modal-loan/ModalLoan.tsx [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+                        title: materialType,
+                        counter: `${list.length}`,
+                      },
+                    ]}
+                  />
+                </div>
+              )}
+              <div className="modal-loan__list__materials">
+                {list.map((listItem, index) => (
+                  <ListMaterials key={index} {...listItem} />
```
Comment (kasperg) [inline src/stories/modal-loan/ModalLoan.tsx:118]: To maintain a good markup structure I suggest making this an (un)ordered list with each material being wrapped by a list item.

File: src/stories/link-filters/LinkFilters.tsx [hunk 5]
```diff
@@ -1,51 +1,33 @@
...
-      {list.map((item) => (
-        <div className="link-filters-tag-wrapper">
-          <Links
-            href={item.href}
-            linkText={item.title}
-            classNames="link-filters-tag"
-          />
+      {props.filters.map((item, index) => (
+        <div key={index} className="link-filters-tag-wrapper">
+          {item.href ? (
```
Comment (kasperg) [inline src/stories/link-filters/LinkFilters.tsx:18]: Could this be an (un)ordered list with list items?

File: src/stories/material-banner/MaterialBanner.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+            </div>
+          )}
+        </div>
+        {props.showBodyText && (
+          <p className="text-body-medium-regular">{props.body}</p>
+        )}
+      </div>
+      <div className="material-banner__cards">
+        {props.cards.map((card, index) => (
+          <MaterialCard key={index} {...card} />
```
Comment (kasperg) [inline src/stories/material-banner/MaterialBanner.tsx:40]: To maintain a good markup structure I suggest making this an (un)ordered list with each material being wrapped by a list item.

File: src/stories/modal-loan/ModalLoan.tsx [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+            size="small"
+            variant="filled"
+            disabled={false}
+            collapsible={true}
+          />
+        </div>
+
+        <div className="modal-loan__list__container">
+          {loanList.map(({ materialType, list }) => (
+            <div className="modal-loan__list">
```
Comment (kasperg) [inline src/stories/modal-loan/ModalLoan.tsx:103]: To maintain a good markup structure I suggest making this an (un)ordered list with each material being wrapped by a list item.

**Review by kasperg — CHANGES_REQUESTED** [review]
Great work! Lots of code, lots of comments :smile:

Due to the size of this PR some changes were quite hard to follow. I have not checked individual commit messages so if my questions are answered there then apologies in advance.

There are a few general themes here:

- **Variables**: I cannot tell what the reasoning is for when to use SCSS variables or not. This especially goes for sizing and sometimes for colors. Sometimes I seem to spot a pattern but variables are not used. Elsewhere a variable is introduced - typically within a specific file - but only used once. 
- **Inline styles**: I expect these should be avoided. If not then I would appreciate soke commenting the places where they are necessary.
- **BEM structure**: I am not sure the suggested changes are in line with the BEM `block__element--modifier` structure that seems to be used elsewhere in the project.
- **Markup structure**: There are places where I have suggested either reducing markup opr updating it to something more semantic (typically lists)

Check out my comments, fix what we can agree on, and then we can discuss how to proceed.

**Comment by matis-dk** [comment]
> There are a few general themes here:
> 
> * **Variables**: I cannot tell what the reasoning is for when to use SCSS variables or not. This especially goes for sizing and sometimes for colors. Sometimes I seem to spot a pattern but variables are not used. Elsewhere a variable is introduced - typically within a specific file - but only used once.

Probably because I'm not following a specific system / practices. Used be cleaned up 👍🏼 


> * **Inline styles**: I expect these should be avoided. If not then I would appreciate soke commenting the places where they are necessary.

Mostly for debugging purposes, but sometimes I forget to change it back to classes.

> * **BEM structure**: I am not sure the suggested changes are in line with the BEM `block__element--modifier` structure that seems to be used elsewhere in the project.

Sorry, lets try change it according to BEM :+1: 

> * **Markup structure**: There are places where I have suggested either reducing markup opr updating it to something more semantic (typically lists)

Check! 

> Check out my comments, fix what we can agree on, and then we can discuss how to proceed.

Yes, thanks Kasper!

File: src/stories/list-details/ListDetails.tsx [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    url: string;
+  };
+};
+
+export const ListDetails = (props: ListDetailsProps) => {
+  return (
+    <div className="list-details">
+      <div className="list-details__icon">
+        <img src={props.icon} alt="arrow accent" />
+      </div>
```
[the remark this answers is not in the document]
Comment (matis-dk) [reply src/stories/list-details/ListDetails.tsx:20]: Nope

File: src/stories/list-reservation/list-reservation.scss [hunk 9]
```diff
@@ -0,0 +1,125 @@
+$space-mobile: 16px;
+$space-desktop: 24px;
```
[the remark this answers is not in the document]
Comment (matis-dk) [reply src/stories/list-reservation/list-reservation.scss:2]: Its just more readable than hardcoded values. We can replace the variables with a comment instead, if you like :)

File: src/stories/material/Material.tsx [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+const placeholderBg = [
+  "bg-identity-tint-120",
+  "bg-identity-tint-80",
+  "bg-identity-tint-60",
+  "bg-identity-tint-40",
+  "bg-identity-tint-20",
+];
+
+export const Material = (props: MaterialProps) => {
+  const bg = placeholderBg[(Math.random() * placeholderBg.length) | 0];
```
[the remark this answers is not in the document]
Comment (matis-dk) [reply src/stories/material/Material.tsx:16]: They would like a random background for each Material card.

File: src/styles/scss/internal.scss (hunk 2 above)
Comment (matis-dk) [reply src/styles/scss/internal.scss:130]: Storybook related classes to setup frame / board

File: src/stories/modal-text/ModalText.tsx [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+          {textContent.map(([title, body]) => (
+            <div className="modal-text__content">
+              <p className="text-body-medium-regular">{title}</p>
+              <p className="text-links">{body}</p>
+            </div>
+          ))}
+        </div>
+      </div>
+      <div className="modal-text__buttons">
+        <div className="modal-text__buttons__inner">
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/modal-text/ModalText.tsx:57]: We can't drop the wrapper in this case. But I have changed it to use ```modal-text__buttons-inner```.

File: src/stories/button-ui/buttons-ui.scss [hunk 12]
```diff
@@ -0,0 +1,8 @@
+$c-btn-border-active: $c-text-primary-black;
+$c-btn-border-disabled: $c-global-tertiary-2;
+
+.button-ui {
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/button-ui/buttons-ui.scss:4]: Changed to ```btn-ui```.

File: src/stories/checkbox/checkbox.scss [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+  user-select: none;
+  cursor: pointer;
+  padding: 6px 8px;
+  border-radius: 6px;
+  overflow: hidden;
+  transition: all 0.3s ease;
+  display: flex;
+}
+
+.checkbox span {
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/checkbox/checkbox.scss:16]: I have added classes for the span tags now.

File: src/stories/checkbox/checkbox.scss [hunk 14]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.checkbox span:first-child {
+  position: relative;
+  flex: 0 0 18px;
+  width: 18px;
+  height: 18px;
+  border-radius: 4px;
+  transform: scale(1);
+  border: 1px solid #cccfdb;
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/checkbox/checkbox.scss:28]: It was the wrong color, I have changed it to use the right variable.

File: src/stories/counter/Counter.tsx [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+      {isReady ? (
+        <img
+          className="counter__icon"
+          src={`icons/basic/icon-check.svg`}
+          alt="check icon"
+        />
+      ) : (
+        <span className="counter__value">{value}</span>
+      )}
+      <span className="counter__label">{label}</span>
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/counter/Counter.tsx:46]: I don't see what is wrong with the naming here? :)

File: src/stories/link-filters/LinkFilters.tsx [hunk 16]
```diff
@@ -1,51 +1,33 @@
...
+            <Links
+              href={item.href}
+              linkText={item.title}
+              classNames="link-filters-tag"
+            />
+          ) : (
+            <span className="text-links uppercase">{item.title}</span>
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/link-filters/LinkFilters.tsx:25]: Made a class for the title which has "uppercase" instead. ```text-links``` is used correctly.

File: src/stories/link-filters/LinkFilters.tsx [hunk 17]
```diff
@@ -1,51 +1,33 @@
...
-            linkText={item.title}
-            classNames="link-filters-tag"
-          />
+      {props.filters.map((item, index) => (
+        <div key={index} className="link-filters-tag-wrapper">
+          {item.href ? (
+            <Links
+              href={item.href}
+              linkText={item.title}
+              classNames="link-filters-tag"
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/link-filters/LinkFilters.tsx:22]: I have changed the structure

File: src/stories/modal-loan/ModalLoan.tsx [hunk 18]
```diff
@@ -0,0 +1,236 @@
...
+            buttonType="default"
+            label={`Forny mulige (${countRenewable})`}
+            size="small"
+            variant="filled"
+            disabled={false}
+            collapsible={true}
+          />
+        </div>
+
+        <div className="modal-loan__list__container">
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/modal-loan/ModalLoan.tsx:101]: 👍

File: src/stories/status-label/StatusLabel.tsx [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+
+export const StatusLabel = (props: StatusLabelProps) => {
+  const { label, status, classNames } = props;
+
+  function getStatus() {
+    if (status === "danger") return "sl-danger";
+    if (status === "warning") return "sl-warning";
+    if (status === "info") return "sl-info";
+    if (status === "neutral") return "sl-neutral";
+    if (status === "outline") return "sl-outline";
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/status-label/StatusLabel.tsx:15]: 👍 fixed

File: src/stories/list-reservation/list-reservation.scss (hunk 9 above)
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/list-reservation/list-reservation.scss:2]: I have made the variable name more specific to only list-reservation.

File: src/stories/list-reservation/list-reservation.scss [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+  @include breakpoint-s {
+    width: 92px;
+    display: block;
+  }
+}
+
+.list-reservation__deadline {
+  display: inline-grid;
+  grid-row-gap: 8px;
+  margin-left: calc(72px + 16px);
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/list-reservation/list-reservation.scss:89]: Nope, I have changed it to 88px.

File: src/stories/list-reservation/list-reservation.scss [hunk 21]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+.list-reservation__status {
+  display: grid;
+  align-items: flex-end;
+  grid-template-columns: auto 1fr auto;
+
+  @include breakpoint-s {
+    align-items: center;
+    // margin-left: $space-desktop;
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/list-reservation/list-reservation.scss:73]: I have removed it 👍

File: src/stories/counter/Counter.tsx [hunk 22]
```diff
@@ -0,0 +1,55 @@
...
+    </div>
+  );
+};
+
+declare global {
+  interface Window {
+    eventHeader: () => void;
+  }
+}
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/counter/Counter.tsx:55]: This function is only used in ```Header.tsx / initheader.js```, so I have removed it in all the other files.

File: src/stories/material-card/material-card.scss [hunk 23]
```diff
@@ -0,0 +1,38 @@
...
+  text-decoration: none;
+}
+
+.material-card__icon {
+  display: flex;
+  justify-content: flex-end;
+
+  svg {
+    cursor: pointer;
+  }
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/material-card/material-card.scss:13]: You are right. I have removed it.

File: src/stories/modal-text/ModalText.tsx [hunk 24]
```diff
@@ -0,0 +1,92 @@
+import { useEffect, useMemo, useState } from "react";
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/modal-text/ModalText.tsx:1]: True, I have removed it.

File: src/styles/scss/shared.scss [hunk 25]
```diff
@@ -22,3 +22,54 @@
...
+  &:hover {
+    transform: rotateZ(90deg);
+  }
+}
+
+.modal {
+  /**
+  * This color is inherited by the <Counter /> component.
+  **/
+  --parent-bg-color: #f6f5f0;
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/styles/scss/shared.scss:49]: No it doesn't. I have changed it to one line.

File: src/styles/scss/shared.scss (hunk 1 above)
Comment (PELN) [reply src/styles/scss/shared.scss:30]: Yup. I have removed the class from ```shared.scss``` and added a new class in ```list-materials.scss``` instead, where it should be used.

File: src/stories/material/Material.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/material/Material.tsx:16]: I have added a comment in the file.

File: src/icons/icon-collection.json [hunk 26]
```diff
@@ -0,0 +1,75 @@
+[
+  "About.svg",
+  "ArrowLeft.svg",
+  "ArrowRight.svg",
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/icons/icon-collection.json:4]: It is to display all the icons from the "collection" folder the same way the other icons are displayed.

I have updated the ```icon-collection.json``` file with ```title``` and ```fileName``` for all icons. And added a stories file + tsx file to display them in the icon folder where the others are.

File: src/stories/checkbox/Checkbox.tsx [hunk 27]
```diff
@@ -0,0 +1,44 @@
...
+      <label className="checkbox" htmlFor={checkboxId.current}>
+        <span>
+          <svg width="20px" height="20px">
+            <polyline
+              points="1.5 6 4.5 9 10.5 1"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="2"
+            />
+          </svg>
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/checkbox/Checkbox.tsx:38]: It is inline because the stroke should be white, which is fixed with css. We only have it in black in ```icons/collection```.
I have added a code comment.

File: src/stories/material-card/MaterialCard.tsx [hunk 28]
```diff
@@ -0,0 +1,56 @@
...
+            <path
+              d="M16.9322 10.3449L16.9283 10.3487L10.1776 17.1758C10.0798 17.2747 9.91991 17.2747 9.82206 17.1758L3.07772 10.354L3.07774 10.354L3.07374 10.35C1.51615 8.79812 0.75 7.03441 0.75 5.36082C0.75 3.95039 1.2289 2.80751 2.01024 2.01966C2.79104 1.23235 3.92287 0.75 5.32246 0.75C6.18981 0.75 6.85479 0.882485 7.43004 1.13946C8.00719 1.39728 8.54059 1.80117 9.11674 2.40989C9.1944 2.49194 9.2735 2.57794 9.35845 2.67265L9.43671 2.76131L9.99303 3.3916L10.556 2.76724L10.6362 2.67825L10.8875 2.40506C11.462 1.79881 11.9941 1.39634 12.5699 1.13922C13.1449 0.882408 13.8098 0.75 14.6771 0.75C16.0767 0.75 17.2087 1.23237 17.9896 2.01971C18.771 2.80759 19.25 3.95046 19.25 5.36082C19.25 7.03465 18.4838 8.79831 16.9322 10.3449Z"
+              strokeWidth="1.5"
+              className={
+                isLiked
+                  ? "bg-identity-primary border-identity-primary"
+                  : "border-secondary-gray"
+              }
+            />
+          </svg>
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/material-card/MaterialCard.tsx:44]: It is. I have added a comment.

File: src/stories/number/Number.tsx [hunk 29]
```diff
@@ -0,0 +1,30 @@
...
+    if (status === "warning") return "number-warning";
+    if (status === "info") return "number-info";
+    if (status === "neutral") return "number-neutral";
+    return "";
+  }
+
+  return (
+    <div className={`number  ${getStatus()}`}>
+      <span>{label}</span>
+    </div>
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/number/Number.tsx:22]: Nope. I have removed it.

File: src/stories/status-label/StatusLabel.tsx [hunk 30]
```diff
@@ -0,0 +1,31 @@
...
+
+    return "";
+  }
+
+  return (
+    <div className={`status-label  ${getStatus()} ${classNames || ""}`}>
+      <span>{label}</span>
+    </div>
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/status-label/StatusLabel.tsx:23]: I have removed it.

File: src/stories/list-reservation/ListReservationsAndLoans.tsx (hunk 3 above)
Comment (PELN) [reply src/stories/list-reservation/ListReservationsAndLoans.tsx:41]: We have decided to use h3 as default, so I have changed it accordingly.

File: src/stories/link-filters/LinkFilters.tsx (hunk 5 above)
Comment (PELN) [reply src/stories/link-filters/LinkFilters.tsx]: Yup. I have changed it.

File: src/stories/modal-loan/ModalLoan.tsx (hunk 4 above)
Comment (PELN) [reply src/stories/modal-loan/ModalLoan.tsx]: 👍 done

File: src/stories/material-banner/MaterialBanner.tsx (hunk 6 above)
Comment (PELN) [reply src/stories/material-banner/MaterialBanner.tsx:40]: 👍 done

File: src/stories/modal-loan/ModalLoan.tsx (hunk 7 above)
Comment (PELN) [reply src/stories/modal-loan/ModalLoan.tsx:103]: 👍 done

File: src/components/ModalShared.tsx [hunk 31]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+export function ModalFallbackButton({
+  toggleModal,
+}: {
+  toggleModal: () => void;
+}) {
+  return (
+    <div
+      style={{ display: "flex", justifyContent: "flex-end", padding: "20px" }}
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/components/ModalShared.tsx:15]: Nope. The styling should be in the shared.scss file. I have moved it now.

File: src/stories/material-banner/MaterialBanner.stories.tsx [hunk 32]
```diff
@@ -0,0 +1,128 @@
...
+      isLiked: false,
+      material: {
+        url: "images/book_cover_1.jpg",
+        animate: true,
+        size: "medium",
+      },
+    },
+    {
+      title: "Lone Als Egebo (2021)",
+      description: "Mikrobiologibogen",
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/material-banner/MaterialBanner.stories.tsx:35]: I have made some more variance in title and description.

There is an issue when the title/desc is too long on smaller screens. I have tried truncating it, but I can't get it to work properly. Maybe @matis-dk could have a look at it? 😊

File: src/stories/modal-details/ModalDetails.tsx [hunk 33]
```diff
@@ -0,0 +1,175 @@
...
+  {
+    title: "Materialenummer",
+    date: "87556608986",
+    icon: "icons/collection/Ebook.svg",
+  },
+];
+
+const materialCards: MaterialCardProps[] = [
+  {
+    title: "Af Isabel Sánchez Vegara",
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/modal-details/ModalDetails.tsx:123]: Also modified this to have more variance.

File: src/stories/modal-details/ModalDetails.stories.tsx [hunk 34]
```diff
@@ -0,0 +1,30 @@
...
+  },
+} as ComponentMeta<typeof ModalDetails>;
+
+const Template: ComponentStory<typeof ModalDetails> = (args) => (
+  <ModalDetails {...args} />
+);
+
+export const Item = Template.bind({});
+Item.args = {
+  title: "Divas, dames & daredevils: Lost Heroines of Golden Age Comi...",
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/modal-details/ModalDetails.stories.tsx:25]: I have added css for truncating the text, since it was hardcoded dots to show the feature.

File: src/stories/material-banner/MaterialBanner.stories.tsx (hunk 32 above)
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/material-banner/MaterialBanner.stories.tsx]: Screenshot 2022-01-26 at 16 17 27

File: src/stories/modal-details/modal-details.scss [hunk 35]
```diff
@@ -0,0 +1,97 @@
...
+
+.modal-details__tags {
+  display: flex;
+  grid-column-gap: 8px;
+  margin-bottom: 16px;
+}
+
+.modal-details__title {
+  display: block;
+  display: -webkit-box;
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/modal-details/modal-details.scss:65]: I have disabled stylelint for this one, since we need it to truncate the text.

File: src/stories/material-banner/material-banner.scss [hunk 36]
```diff
@@ -0,0 +1,77 @@
+.material-banner__header {
+  padding: 24px;
+
+  @include breakpoint-s {
+    padding: 24px 45px;
+  }
+
+  @include breakpoint-m {
+    padding-left: 100px;
+    padding: 51px 45px;
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/material-banner/material-banner.scss]: I have changed it to one line now.

File: src/stories/modal-details/modal-details.scss [hunk 37]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.modal-details__tags {
+  display: flex;
+  grid-column-gap: 8px;
+  margin-bottom: 16px;
+}
+
+/* stylelint-disable value-no-vendor-prefix */
+/*
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/modal-details/modal-details.scss:56]: Done

File: src/stories/material-card/material-card.scss [hunk 38]
```diff
@@ -0,0 +1,40 @@
...
+  display: grid;
+
+  @include breakpoint-s {
+    margin-top: 32px;
+  }
+}
+
+.material-card__description {
+  @extend %text-body-small-regular;
+  white-space: nowrap;
```
[the remark this answers is not in the document]
Comment (PELN) [reply src/stories/material-card/material-card.scss:33]: done..

File: src/stories/material-banner/MaterialBanner.stories.tsx (hunk 32 above)
[the remark this answers is not in the document]
Comment (matis-dk) [reply src/stories/material-banner/MaterialBanner.stories.tsx]: We solved it ;)

File: src/stories/list-reservation/ListReservationsAndLoans.tsx [hunk 39]
```diff
@@ -0,0 +1,116 @@
...
+
+          {fee > 0 ? (
+            <div className="list-reservation__fee">
+              <p className="text-body-medium-medium">Gebyr {fee},-</p>
+            </div>
+          ) : null}
+        </div>
+      </div>
+      {isStacked && <div className="list-reservation__stacked_1" />}
+      {isStacked && <div className="list-reservation__stacked_2" />}
```
[the remark this answers is not in the document]
Comment (matis-dk) [reply src/stories/list-reservation/ListReservationsAndLoans.tsx]: Yes, I replaced them with the ::after and ::before pseudo elements :+1:

File: src/stories/modal-details/modal-details.scss [hunk 40]
```diff
@@ -0,0 +1,104 @@
...
+/* 
+  We disable stylelint here since -webkit-box 
+  uses ellipsis for truncating which we need.
+  https://caniuse.com/?search=line-clamp
+*/
+
+/* stylelint-disable value-no-vendor-prefix */
+.modal-details__title {
+  display: block;
+  display: -webkit-box;
```
Comment (kasperg) [inline src/stories/modal-details/modal-details.scss:65]: @PELN: This effectively disables the `value-no-vendor-prefix` rule for the rest of the file. I do not think that was what you intended :smile:. [Please use `stylelint-disable-next-line` instead](https://stylelint.io/user-guide/ignore-code/).

Comment (PELN) [reply src/stories/modal-details/modal-details.scss]: I have changed it now :)
